### DISCUSSION TITLE
Top panel: via point + name of next street

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#383] [Q&A] [answered]
**Cruiser current.**
Top panel: Name of next street. (Set) 
@ Turnpoints > Display: "**Streetname**"
@ Viapoints > Display "**Name**". (Streetname missing)

**Suggestion new**.
Show both via Name & Streetname.
@ Viapoints > Display: "**Name | Streetname**"

Example by a gpx via navtrkpt "name & cmt & sym & type"

![Via trackpoint name -streetname](https://github.com/devemux86/cruiser/assets/54167201/ae21ddb1-2574-46f8-b9a1-7d6ea66fc89c)
![Via checkpoint   u-turn](https://github.com/devemux86/cruiser/assets/54167201/0f8dffe0-2d61-4c7d-bb42-dd5bd826a57c)

![Via name   Turn_Icon   Streetname](https://github.com/devemux86/cruiser/assets/54167201/65248090-8b5f-44fb-9aa7-dd948409ad9a)

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi thanks for the suggestion.

Do you mean the panel "Name of next street"?

- It displays the street name of the next turn instruction.
- In case of next via point, it displays the via point name.

So you suggest in the 2nd case to display both via point name + street name?
